### ISSUE TITLE
cargo make ci  failed

### ISSUE THREAD

**Issue by stappersg** [tui-rs-revival/ratatui#315]
Hello,

While preparing #314 there was _build error_.

## Description

```text
[cargo-make] INFO - Execute Command: "cargo" "clippy" "--all-targets" "--no-default-features" "--tests" "--benches" "--features" "serde,crossterm" "--" "-D" "warnings"
    Checking ratatui v0.21.0 (/home/stappers/src/rust/ratatui)
error: using `.borrow()` on a double reference, which returns `&text::masked::Masked<'_>` instead of borrowing the inner type
   --> src/text/masked.rs:116:32
    |
116 |         let text: Text = masked.borrow().into();
    |                                ^^^^^^^^^
    |
    = note: `-D suspicious-double-ref-op` implied by `-D warnings`

error: using `.borrow()` on a double reference, which returns `&text::masked::Masked<'_>` instead of borrowing the inner type
   --> src/text/masked.rs:122:35
    |
122 |         let cow: Cow<str> = masked.borrow().into();
    |                                   ^^^^^^^^^

error: could not compile `ratatui` (lib test) due to 2 previous errors
[cargo-make] ERROR - Error while executing command, exit code: 101
[cargo-make] WARN - Build Failed.
error: failed to push some refs to 'github.com:stappersg/ratatui.git'
```

## To Reproduce

Probably `cargo make ci`

## Expected behavior

Smooth `git push`

## Additional context

Used workaround `git push --no-verify`

**Comment by joshka** [comment]
This was fixed in https://github.com/tui-rs-revival/ratatui/pull/311 - You should be able to pull main and rebase to address the issue.

**Comment by stappersg** [comment]
> This was fixed in #311 - You should be able to pull main and rebase to address the issue.

Yes, _git pull --rebase_ could / would have avoid this issue.
